pkg/oc/bootstrap/docker/up.go

```diff
@@ -304,6 +304,15 @@ func (c *ClusterUpConfig) Complete(f *osclientcmd.Factory, cmd *cobra.Command, o
 	c.dockerClient = client
 	taskPrinter.Success()
 
+	// Ensure that the OpenShift Docker image is available.
+	// If not present, pull it.
+	// We do this here because the image is used in the next step if running Red Hat docker.
+	taskPrinter.StartTask(fmt.Sprintf("Checking if image %s is available", c.openshiftImage()))
+	if err := c.checkOpenShiftImage(out); err != nil {
```

Comment:
This was previously called in `Check`.  What needs it earlier?

Reply:
addressed in comment above.